wrappers/nodejs/index.js

```diff
@@ -2280,36 +2297,57 @@ class FrameSet {
     }
   }
 
-  __internalGetFrame(stream) {
-    return Frame._internalCreateFrame(this.cxxFrameSet.getFrame(stream));
+  __internalGetFrame(stream, streamIndex) {
+    return Frame._internalCreateFrame(this.cxxFrameSet.getFrame(stream, streamIndex));
+  }
+
+  __internalFindFrameInCache(stream, streamIndex) {
+    for (let i = 0; i < this.cacheMetadata.length; i++) {
```

Comment:
how about use for-of instead， which is more preferred in ES6

Reply:
yes, we can use for (const [i, data] of this.cacheMeatdata.entries()) to get both the data and the index.